plugins/woocommerce-admin/client/marketplace/components/woo-connect-plugin/plugin-install-notice.tsx

```diff
@@ -34,10 +33,10 @@ export default function PluginInstallNotice() {
 					{ message }
 					<div className="components-notice__buttons">
 						<Button
-							href={ WOO_CONNECT_PLUGIN_INSTALL_URL }
+							href={ wccomSettings?.wooConnectInstallUrl }
```

Comment:
If the plugin name is Woo Update Manager, how about changing the variable name to reflect that?

Reply:
fixed with https://github.com/woocommerce/woocommerce/pull/44279/commits/34572cd17d9bb7a4aa97484d1f8fc2cb9753388a